### ISSUE TITLE
Consider updating the fractional gates tutorial

### ISSUE THREAD

**Issue by yaelbh** [Qiskit/documentation#2479]
Regarding this tutorial: https://github.com/Qiskit/documentation/blob/main/docs/guides/fractional-gates.ipynb
We recently added a new feature: transpilation of circuits with a backend that supports fractional gates will automatically transform the circuits, such that all `rzz` numeric angles are in the valid range of `0 - pi/2`.

I think we should add a comment about it in the tutorial, because:
1. Users may be confused that their circuit ran successfully, in spite of invalid angles therein.
2. To inform users about a modification of their circuit during transpilation.

But I also hesitate, having the concern that this will make the tutorial too detailed and complex. In particular note that the transformation applies to numeric angles but not to parametric ones.

Discussion a bit of discussion in https://github.com/Qiskit/qiskit-ibm-runtime/pull/2043.

cc: @nkanazawa1989 @wshanks @kaelynj

**Comment by wshanks** [comment]
I agree it would be good to mention that the qiskit-ibm-runtime transpiler plugin can handle the rzz angles when they are numeric, but I would rather let the docs team decide where that level of detail goes. Maybe it could be a footnote on the angle range given at the beginning of the document. Or maybe it could be a section near the end or in a separate linked document if there is another document on transpilation that is more appropriate.
